arm:
  type: ur10
sensors:
  logical_camera_1:
    type: logical_camera
    pose:
      xyz: [1.2, 3.40, 1.56]
      rpy: ['pi/2', 1.567591, 1.567635]

  break_beam_1:
    type: break_beam
    pose:
      xyz: [1.60, 0.2, 0.91]
      rpy: [0, 0, 'pi']

  break_beam_2:
    type: break_beam
    pose:
      xyz: [1.60, 0.85, 0.91]
      rpy: [0, 0, 'pi']


  break_beam_3:
    type: break_beam
    pose:
      xyz: [1.60, 1.50, 0.91]
      rpy: [0, 0, 'pi']

  break_beam_4:
    type: break_beam
    pose:
      xyz: [1.60, 2.15, 0.91]
      rpy: [0, 0, 'pi']
